vendor/github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared/networkaddonsconfig_types.go

```diff
@@ -17,6 +17,7 @@ type NetworkAddonsConfigSpec struct {
 	ImagePullPolicy        corev1.PullPolicy         `json:"imagePullPolicy,omitempty"`
 	NMState                *NMState                  `json:"nmstate,omitempty"`
 	KubeSecondaryDNS       *KubeSecondaryDNS         `json:"kubeSecondaryDNS,omitempty"`
+	KubevirtIpamController *KubevirtIpamController   `json:"kubevirtIpamController,omitempty"`
```

Comment:
CNAO need to be vendored once there is a new release, and then this file wont be needed
until then this PR is on hold